### PR TITLE
Update iOS push-notifications.md to use Amplify CLI flow

### PR DESCRIPTION
**Pull request by lawmicha**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lawmicha** [comment]
need to check v2-preview if there is a notifications section that needs to be updated

**Comment by lawmicha** [comment]
Hi @UnleashedMind - has the changes for configuring notifications been released in Amplify CLI? can this PR be merged?

@rjuliano  FYI this is the PR that contains doc updates to the iOS PN. the bottom contains creating campaign instructions and should be in sync with android, and both iOS/Android needs journeys

File: sdk/ios/push-notifications.md [hunk 1]
```diff
@@ -176,110 +243,46 @@ Make the following changes to the configuration file. The values are available i
...
-    ![Image of turning on Push Notifications capabilities in Xcode]({{image_base}}/xcode-turn-on-push-notification.png)
-
-1. Configure the app to run in the **Release** profile instead of the default **Debug** profile. Perform the following steps to get a notification to the device:
-
-    1. For your app target, go to the **General** tab of project configuration and make sure **Automatically Manage Signing** check box is not selected.
-
-    1. In the **Signing (Release)** section, choose the production provisioning profile you created on Apple developer console. For testing push notifications on a device, you will need an [Ad Hoc Provisioning Profile](https://help.apple.com/xcode/mac/current/#/dev4335bfd3d) configured with a Production AppStore and Ad Hoc certificate, and with the device(s) to be used for testing.
+```
+? Choose the push notification channel to enable. `APNS`
+? Choose authentication method used for APNs `Key`
```
Comment (UnleashedMind) [inline sdk/ios/push-notifications.md:258]: The user can also chose to use `Certificate` as the authentication method here, which uses a `.p12` file. 
I think we should also add its workflow here.

File: sdk/ios/push-notifications.md [hunk 2]
```diff
@@ -176,110 +243,46 @@ Make the following changes to the configuration file. The values are available i
...
+? The bundle id used for APNs Tokens:  <Your App's BundleID like com.yourname.projectname>
+? The team id used for APNs Tokens:  `XXXXXXXXX`
+? The key id used for APNs Tokens:  `ABCDEXXXXX`
+? The key file path (.p8):  `AuthKey_ABCDEXXXXX.p8`
+✔ The APNS channel has been successfully enabled.
+```
 
-    1. In the top left corner of Xcode (where your app name is displayed next to the current build target device), choose on your app name and then select **Edit Scheme**, and then set **Build configuration** to **Release**
+4. Open the AWS Pinpoint console with `amplify console analytics`
```
Comment (UnleashedMind) [inline sdk/ios/push-notifications.md:266]: You can open the console by directly run `amplify notification console`

**Pull request closed without merging**